x-pack/test/saml_api_integration/apis/security/saml_login.ts

```diff
@@ -443,7 +443,7 @@ export default function({ getService }: FtrProviderContext) {
       it('should invalidate access token on IdP initiated logout', async () => {
         const logoutRequest = await createLogoutRequest({ sessionIndex: idpSessionIndex });
         const logoutResponse = await supertest
-          .get(`/api/security/logout?${querystring.stringify(logoutRequest)}`)
+          .get(`/api/security/logout?${stringify(logoutRequest, { sort: false })}`)
```

Comment:
In other places if seems you replaced stringing with url.stringifyUrlQuery. why isn't this the case here?

Reply:
You are right, I did it in almost all places instead `functional tests` and  `kibana-core` code. Not sure that we should add import to 'kibana_utils' from that places.